Library/Formula/libpng.rb

```diff
@@ -28,6 +28,7 @@ class Libpng < Formula
 
   def install
     ENV.universal_binary if build.universal?
+    system "autoreconf", "--force", "--install" if head?
```

Comment:
You can use `system "autoreconf", "-fvi"` rather than explicitly writing it out. Should be `if build.head?` as well.